platforms/documentation/docs/src/docs/userguide/authoring-builds/gradle-properties/service_injection.adoc

```diff
@@ -226,15 +210,7 @@ The `ArchiveOperations` service is injected into the `MyArchiveOperationsTask` t
 == 8. `ExecOperations`
```

Comment:
❌ The next section is missing the dot after the number, let's fix that as well

Reply:
Done, thanks for noticing